### PR TITLE
added tron to basemapstyle

### PR DESCRIPTION
**Pull request by hanbyul-here**
- Added tron styles to BasemapStyles
- updated examples

### CODE REVIEW

**Comment by migurski** [comment]
The documentation should be updated as well.

**Comment by hanbyul-here** [comment]
+1: updated

**Comment by rmglennon** [comment]
ok, i'll do the docs quickly and we can bundle it all in a release

**Comment by rmglennon** [comment]
oh, nevermind! done. These have .zip extensions now?

File: src/js/components/basemapStyles.js [hunk 1]
```diff
@@ -12,7 +12,10 @@ var style = {
   ZincNoLabels: 'https://mapzen.com/carto/zinc-style-no-labels/3/zinc-style-no-labels.yaml',
   Walkabout: 'https://mapzen.com/carto/walkabout-style/1/walkabout-style.yaml',
   WalkaboutMoreLabels: 'https://mapzen.com/carto/walkabout-style-more-labels/1/walkabout-style-more-labels.yaml',
-  WalkaboutNoLabels: 'https://mapzen.com/carto/walkabout-style-no-labels/1/walkabout-style-no-labels.yaml'
+  WalkaboutNoLabels: 'https://mapzen.com/carto/walkabout-style-no-labels/1/walkabout-style-no-labels.yaml',
+  Tron: 'https://mapzen.com/carto/tron-style/tron-style.zip',
```
Comment (nvkelso) [inline src/js/components/basemapStyles.js]: These need to be versioned, like:

```
Tron: 'https://mapzen.com/carto/tron-style/2/tron-style.zip',
TronMoreLabels: 'https://mapzen.com/carto/tron-style-more-labels/2/tron-style-more-labels.zip',
TronNoLabels: 'https://mapzen.com/carto/tron-style-no-labels/2/tron-style-no-labels.zip'
```

**Comment by hanbyul-here** [comment]
@rmglennon seems that so? Idk much about it ! @bcamper or @meetar probably know better

**Comment by meetar** [comment]
@rmglennon Are you asking about Tangram's ability to load a bundled zip, or about the house styles being offered as zips?

**Comment by bcamper** [comment]
@rmglennon Tangram can load styles from either a `.yaml` or a `.zip`. Tron 2.0 is the first house style we are publishing as a `.zip` though (we may offer zips of others when we do some refactoring).

**Comment by rmglennon** [comment]
got it, all good. Thanks @meetar and @bcamper. Was just verifying because these changes have URLs with zips for the file extensions, vs the familiar yaml!

**Pull request merged**